aiven/client/client.py

```diff
@@ -1397,6 +1398,8 @@ def create_project(
             body["billing_emails"] = [{"email": email} for email in billing_emails]
         if tech_emails is not None:
             body["tech_emails"] = [{"email": email} for email in tech_emails]
+        if use_source_project_billing_group is not None:
+            body["use_source_project_billing_group"] = True
```

Comment:
Doesn't change anything in that case but it should be `body["use_source_project_billing_group"] = use_source_project_billing_group`

Reply:
Fixed